clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/provider/agent/Namespace.groovy

```diff
@@ -49,4 +49,8 @@ enum Namespace {
   String toString() {
     ns
   }
+
+  String getNs() {
+    return ns
+  }
```

Comment:
...And so this method is also unnecessary.

Reply:
Also java + groovy fun, the ecs provider breaks w/o this explicitly declared. Won't compile without this method :(